src/parameters/parameters_flow_solver.cpp

```diff
@@ -408,6 +425,13 @@ void FlowSolverParam::parse_parameters(dealii::ParameterHandler &prm)
           number_of_times_to_output_velocity_field = get_number_of_values_in_string(output_velocity_field_times_string);
           output_vorticity_magnitude_field_in_addition_to_velocity = prm.get_bool("output_vorticity_magnitude_field_in_addition_to_velocity");
           output_flow_field_files_directory_name = prm.get("output_flow_field_files_directory_name");
+            // Check if directory exists - see https://stackoverflow.com/a/18101042
+            struct stat info;
```

Comment:
Resolve `struct stat info;` shadowing local variable. 
https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=244&id=dougshidong_PHiLiP&open=AYr7U1ps4kO947chs77C

Reply:
Fixed, thanks for highlighting that.